configure.ac

```diff
@@ -6819,7 +6833,7 @@ if test "$ENABLED_SP_ASM" = "yes" && test "$ENABLED_SP" = "yes"; then
     ENABLED_SP_X86_64_ASM=yes
     if test "x$ENABLED_FASTMATH" = "xno"
     then
-      AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_X86_64_BUILD"
+      AM_CFLAGS="$AM_CFLAGS"
```

Comment:
Is this necessary?

Reply:
Probably not but I wanted this to always to be defined.
The C code calling the AES ASM was going to need to know but it doesn't anymore.
May be required after future x86 ASM but not right now.